File: 02_Object_Oriented_Programming/03_Constructor_and_It's_Types/01_Default_Constructor/python/movie.py
class Movie:
    # Custom default constructor
    def __init__(self):
        self.title = "Untitled"  # Default: "Untitled"
        self.duration = 90       # Default: 90

    def display_details(self):
        print(f"Title: {self.title}, Duration: {self.duration} mins")

File: 02_Object_Oriented_Programming/03_Constructor_and_It's_Types/01_Default_Constructor/javascript/movie.js
class Movie {
    // Custom default constructor
    constructor() {
        this.title = "Untitled"; // Default: "Untitled"
        this.duration = 90;      // Default: 90
    }

    displayDetails() {
        console.log(`Title: ${this.title}, Duration: ${this.duration} mins`);
    }
}

module.exports = Movie;

